app/models/miq_ae_method.rb

```diff
@@ -14,6 +14,14 @@ class MiqAeMethod < ApplicationRecord
   validates_format_of     :name, :with    => /\A[\w]+\z/i,
                                  :message => N_("may contain only alphanumeric and _ characters")
 
+  ANSIBLE_JOB_TEMPLATE_SYMBOLS = [:repository_id,
+                                  :playbook_id,
+                                  :credential_id,
+                                  :network_credential_id,
+                                  :cloud_credential_id,
+                                  :verbosity,
+                                  :become_enabled].freeze
```

Comment:
Should we also be doing a validation here when the `language` is `playbook` instead of waiting until runtime or replying on the UI.  This would catch invalid imports as well.

Reply:
Can that be a different  PR? This is just adding a constant we could use. I am missing :extra_vars which I need to add.

Reply:
@gmcculloug 
Did you mean location or language?
Location already contains the playbook
AVAILABLE_LOCATIONS = %w(builtin inline uri expression playbook).freeze

Reply:
Meant `location `.  I guess it can be a different PR, I didn't think it would be much extra work to add it here.  Let me know.